README.md

```diff
@@ -5,38 +5,38 @@ We—the undersigned—advocate a change to the approach that web standards comm
-* The web platform should develop, describe and test new high-level features in JavaScript, and allow web developers to iterate on them before they become standardized. This creates a ***_virtuous cycle_*** between standards and developers.
-* The standards process should ***_prioritize efforts_*** that follow these recommendations and deprioritize and refocus those which do not.
+* The web platform should develop, describe, and test new high-level features in JavaScript, while allowing web developers to iterate on them before they become standardized. This creates a ***_virtuous cycle_*** between standards and developers.
+* The standards process should ***_prioritize efforts_*** that follow these recommendations; efforts that do not should be deprioritized and refocused.
 
 ------
 
 By focusing on standardizing ***_new low-level capabilities_***, and building new features in terms of them, we:
 * Contain new security surface area.
-* Allow optimizations in browser engines to focus on the stable core, which affects more APIs as they are added. This leads to better performance with less implementation effort.
+* Allow optimizations in browser engines to focus on the stable core, which during implementation can positively affect other APIs', leading to better performance with less implementation effort.
 * Allow browser vendors and library authors to iterate on libraries that provide developer-friendly, high-level APIs.
 
 By explaining existing and new features ***_in terms of low-level capabilities_***, we:
 * Reduce the rate of growth in complexity, and therefore bugs, in implementations.
 * Make it possible to polyfill more of the platform's new features.
-* Require less developer education for new features. Educational materials can build off of concepts that are already in the platform.
+* Require less developer education for new features. Educational materials can build off of concepts already present in the platform.
 
 Making new features easy to understand and polyfill introduces a ***_virtuous cycle_***:
 * Developers can ramp up more quickly on new APIs, providing quicker feedback to the platform while the APIs are still the most malleable.
-* Mistakes in APIs can be corrected quickly by the developers who use them, and library authors who serve them, providing high-fidelity, critical feedback to browser vendors and platform designers.
-* Library authors can experiment with new APIs and create more cow-paths for the platform to pave.
+* Mistakes in APIs can be corrected quickly by the developers who use them and library authors who serve them, providing high-fidelity, critical feedback to browser vendors and platform designers.
+* Library authors can experiment with new APIs and create more ["cow paths"](https://en.wikipedia.org/wiki/Desire_path) for the platform to pave.
```

Comment:
using `en` prefix because Wikipedia can possibly show a 404 if the doc is not localised (to my knowledge): https://wikipedia.org/wiki/Desire_path